airbyte-server/build.gradle

```diff
@@ -44,7 +44,7 @@ publishing {
 }
 
 dependencies {
-    implementation 'io.temporal:temporal-sdk:1.0.4'
+    implementation 'io.temporal:temporal-sdk:1.6.0'
```

Comment:
I am also doing it in another PR, I switch to 1.5 because the 1.6 is not publish to maven central https://mvnrepository.com/artifact/io.temporal/temporal-sdk, Does the 1.6 exist?

Reply:
Weird, it's able to pull it. Not sure why it isn't showing yet.

Reply:
They released it https://github.com/temporalio/sdk-java/releases/tag/v1.6.0